src/test/kotlin/net/maizegenetics/phgv2/cli/ExportHvcfTest.kt

```diff
@@ -0,0 +1,104 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.testing.test
+import com.google.common.io.Files
+import net.maizegenetics.phgv2.utils.getChecksum
+import org.apache.logging.log4j.LogManager
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.io.File
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+@ExtendWith(TestExtension::class)
+class ExportHvcfTest {
+
+    companion object {
+
+        private val myLogger = LogManager.getLogger(ExportHvcfTest::class.java)
+
+        val exportHvcfDir = "${TestExtension.tempDir}/export-vcfs/"
+        val outputHvcfDir = "${exportHvcfDir}/output/"
+        val inputHvcfDir = "${exportHvcfDir}/input/"
+        val dbPath = "${exportHvcfDir}/tiledb_export_hvcf"
+        val testHvcfFile = inputHvcfDir + "/" + File(TestExtension.smallseqRefHvcfFile).name
+
+        @BeforeAll
+        @JvmStatic
+        fun setup() {
+
+            File(exportHvcfDir).mkdirs()
+            File(outputHvcfDir).mkdirs()
+            File(inputHvcfDir).mkdirs()
+
+            Initdb().createDataSets(dbPath)
+
+            Files.copy(File(TestExtension.smallseqRefHvcfFile), File(testHvcfFile))
+
+            var builder = ProcessBuilder("conda", "run", "-n", "phgv2-conda", "bgzip", testHvcfFile)
+            var redirectError = "$exportHvcfDir/bgzip_error.log"
+            var redirectOutput = "$exportHvcfDir/bgzip_output.log"
+            builder.redirectOutput(File(redirectOutput))
+            builder.redirectError(File(redirectError))
+
+            println("bgzip Command: " + builder.command().stream().collect(Collectors.joining(" ")));
```

Comment:
Also consider removing the stream() here as well and below in this class.

Reply:
Changed